README.md

```diff
@@ -523,6 +524,11 @@ mode = "once"
 essential = true
 ```
 
+##### Mount propagations in bootstrap and superpowered containers
+Both bootstrap and superpowered host containers are configured with a bind mount that points to `/mnt` in the host, which itself is a bind mount of `/local/mnt`.
```

Comment:
I think it makes more sense for the bind mount to be mounted at `/.bottlerocket/rootfs/mnt` inside the container.  That way all of the host-related configuration conceptually happens inside the same hierarchy.